docs/community/contributor_ladder.mdx

```diff
@@ -0,0 +1,191 @@
+a group of related repositories, 
+a service (e.g., a website), 
+or subproject to support the other subprojects (e.g., marketing or community management).
+
+Changes in ORAS Org owners have to be announced via an 
+[issue on the Community repository](https://github.com/oras-project/community/issues). 
+Changes to sub-project owners are to be announced via the appropriate sub-project issue.
+
+### Owner Responsibility
+ORAS owners adhere to the requirements and responsibilities set forth in the respective ORAS Org Owners and Subproject Owners. 
+They further pledge the following:
+
+* To act in the best interest of the project and subprojects.
+* To ensure that project and subproject development and direction is a function of the needs of the community.
+* To take action when reasonably confident that it is the right action to take.
+* To fulfill the responsibilities outlined in this document and its dependents.
+
+### ORAS Org Owners
+The ORAS Org owners are responsible for:
+
+* Maintaining the mission, vision, values, and scope of the project.
+* Refining the governance and charter as needed.
+* Making project level decisions.
+* Resolving escalated project decisions when the subproject owners responsible are blocked.
+* Managing the ORAS brand.
+* Controlling access to ORAS assets, such as source repositories, hosting, project calendars.
+* Deciding what subprojects are part of the ORAS project.
+* Deciding on the creation of new subprojects.
+* Overseeing the resolution and disclosure of security issues.
+* Managing financial decisions related to the project.
+
+Changes to org owners abide by the following:
+
+* Any subproject owner is eligible for a position as an org owner.
+* No one company or organization can employ a simple majority of the org owners.
+* An org owner may step down by submitting an issue stating their intent and they will be moved to emeritus.
+* Org owners MUST remain active on the project. 
+If they are unresponsive for > 6 months, 
+they will lose org ownership unless a super-majority of the other org owners agrees to extend the period to be greater than 6 months.
+* When there is an opening for a new org owner, 
+any current owners may nominate a suitable subproject owners as a replacement
+    * Nominations for new owners must be made by creating an issue.
+* When nominated individual(s) agrees to be a candidate for ownership, 
+the subproject owners may vote
+    * The voting period will be open for a minimum of three business days and 
+    will remain open until a super-majority of project owners has voted.
+    * Only current org owners are eligible to vote via casting a single vote each via a +1 comment on the nomination issue.
+    * Once a super-majority has been reached the owner elect must complete onboarding prior to becoming an official ORAS owner.
+    * Once the owner onboarding has been completed, a pull request is made on the repo adding the new owner to the OWNERS file.
```

Comment:
Generally should avoid org and repo here in favor of organization and repository